src/main/java/htsjdk/variant/variantcontext/writer/VariantContextWriter.java

```diff
@@ -49,4 +55,13 @@
     public boolean checkError();
     
     public void add(VariantContext vc);
+
+    /**
```

Comment:
It seems like the javadoc should mention that you should call exactly one of `writeHeader()` / `setHeader()`